flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -469,21 +479,84 @@ public void sqlUpdate(String stmt) {
 				createTableOperation.getCatalogTable(),
 				createTableOperation.getTableIdentifier(),
 				createTableOperation.isIgnoreIfExists());
+		} else if (operation instanceof CreateDatabaseOperation) {
+			CreateDatabaseOperation createDatabaseOperation = (CreateDatabaseOperation) operation;
+			Catalog catalog = getCatalogOrThrowException(createDatabaseOperation.getCatalogName());
+			String exMsg = getDDLOpExecuteErrorMsg("CREATE DATABASE", createDatabaseOperation.getCatalogName());
```

Comment:
replace `CREATE DATABASE` with `createDatabaseOperation.asSummaryString`? And you don't need provide catalog name anymore